packages/grafana-ui/src/components/ValueMappingsEditor/ValueMappingsEditor.tsx

```diff
@@ -18,7 +18,7 @@ export class ValueMappingsEditor extends PureComponent<Props, State> {
   constructor(props: Props) {
     super(props);
 
-    const mappings = props.valueMappings;
+    const mappings = props.valueMappings || [];
```

Comment:
I got an error while testing this PR but have not been able to reproduce on master.  The is likely caused by #17043

Reply:
merged in #18092